pkg/sql/executor_ir_test.go

```diff
@@ -263,3 +264,23 @@ func TestLogChanWriter_Write(t *testing.T) {
 	_, more := <-c
 	a.False(more)
 }
+
+func TestRewriteStatementsWithHints4Alisa(t *testing.T) {
+	if os.Getenv("SQLFLOW_submitter") != "alisa" {
+		t.Skip("Skip test case: submitter is not alisa.")
+	}
+	dialect := "alisa"
+	hint1, hint2 := `set odps.stage.mapper.num=1;`, `set odps.sql.mapper.split.size=4096;`
+	standardSQL, extendedSQL := `select 1;`, `select 1 to predict d.t.f using m;`
+	sqlProgram := hint1 + standardSQL + hint2 + extendedSQL
```

Comment:
How about adding one or more `\n`s and whitespace and `\t`s in between the statements?

Reply:
Done.